assets/component-menu-drawer.css

```diff
@@ -217,7 +217,7 @@ details[open].menu-opening > .menu-drawer__submenu {
 
 @media screen and (max-width: 749px) {
   .header--has-account:where(:not(.header--has-social):not(.header--has-localizations)) .menu-drawer__utility-links {
-    padding: 2rem;
+    padding: 3rem;
```

Comment:
In other scenarios we're using `2rem 3rem`. Maybe we could do the same here as I don't think it needs the extra `1rem` of padding at the top/bottom.

Reply:
Same below on line `226`. Maybe we can confirm with a designer wether we should apply a consistent `2rem 3rem` padding or sometimes have `3rem` and sometimes not 🤔

Reply:
these are really good suggestions! And a good point to bring up! I am updating the code to keep it consistent